src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -523,13 +522,9 @@ protected virtual void Dispose(bool finalize)
 
```

Comment:
GitHub won't let me leave a comment the line above this, but an easy follow up to this PR is the this.Services.GetService<IWorkspaceEventListenerService>()?.Stop(); line and making that also be disposable based.

Reply:
This is likely already part of #48068, but will verify.